device/common/CMakeLists.txt

```diff
@@ -39,6 +39,11 @@ traccc_add_library( traccc_device_common device_common TYPE SHARED
    "include/traccc/seeding/device/make_doublet_counter_buffer.hpp"
    "src/seeding/make_doublet_counter_buffer.cpp"
    "include/traccc/seeding/device/make_doublet_buffers.hpp"
-   "src/seeding/make_doublet_buffers.cpp" )
+   "src/seeding/make_doublet_buffers.cpp"
+   "include/traccc/seeding/device/count_triplets.hpp"
+   "include/traccc/seeding/device/impl/count_triplets.ipp"
+   "include/traccc/edm/device/triplet_counter.hpp"
```

Comment:
Just to silence my OCD, please move the `triplet_counter.hpp` line up a bit, just after the `doublet_counter.hpp` line. :wink: